tests/Avalonia.UnitTests/MouseTestHelper.cs

```diff
@@ -104,7 +104,18 @@ public void Move(Interactive target, Interactive source, in Point position, KeyM
             Down(target, source, button, position, modifiers);
             Up(target, source, button, position, modifiers);
         }
-        
+
+        public void DoubleClick(Interactive target, MouseButton button = MouseButton.Left, Point position = default,
+            KeyModifiers modifiers = default)
+            => DoubleClick(target, target, button, position, modifiers);
+        public void DoubleClick(Interactive target, Interactive source, MouseButton button = MouseButton.Left,
```

Comment:
nit: Missing blank line above

Reply:
I did this to follow the existing Click methods. Would you prefer I fix that spacing too in this PR?

Reply:
Personally, I would always add the blank lines or the vast majority of standard formatting rules throw warnings. Usually, I make small formatting fixes to related areas of code together. Everyone is different though and there are special cases.

Reply:
Addressed, and fixed other lines in this file.

Reply:
Great!